Source/DayView.swift

```diff
@@ -60,7 +60,22 @@ public final class DayView: UIView, TimelinePagerViewDelegate {
     }
   }
 
-  public var calendar: Calendar = Calendar.autoupdatingCurrent
+  public var calendar: Calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // TODO: Should we observe calendar in state rather then change it manualy?
+      dayHeaderView.calendar = self.calendar
+      timelinePagerView.calendar = self.calendar
+
+      let date = self.state?.selectedDate ?? Date()
```

Comment:
If possible, please omit `self`. I suggest quickly looking thru [CONTRIBUTING.md](https://github.com/richardtop/CalendarKit/blob/master/CONTRIBUTING.md) with the code style examples. Not a requirement, I could fix those issues myself, after your pull request gets merged. Good to keep the library in a common style.